magpy/absolutes.py

```diff
@@ -1494,6 +1496,7 @@ def absoluteAnalysis(absdata, variodata, scalardata, **kwargs):
         - expI:         (float) expected Inclination - failure produced when I differs by more than expT deg
         - expT:         (float) expected value threshold - default 1 deg
         - movetoarchive:(string) define a local directory to store archived data (only works when reading files)
+        - usgsstream:   ((DataStream object) provides vario and scalar data as an alternative to reading files
```

Comment:
This describes the parameter as `usgsstream`, but kwargs.get below uses `usgsdata`...  Can this be a more generic parameter like `datastream`?